graphql/schema.graphql

```diff
@@ -915,6 +915,7 @@ type Annotation {
   note: Note
   issues: [IssueLink]
   suspectedIssues: [IssueLink]
+  userModifyPermission: Boolean!
```

Comment:
nit: can we change this `canModify` , `hasModifyPermission` or another label that implies a boolean?

Reply:
I want it to be clear that this holds the value for the current user. canModify makes it sound like we are saying that this annotation can be modified. I can change it to UserHasModifyPermission. Is that too long?

Reply:
I liked `userCanModify` more